ui/app/components/task-row.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+import { task, timeout } from 'ember-concurrency';
+import { lazyClick } from '../helpers/lazy-click';
+
+export default Component.extend({
+  store: service(),
+  token: service(),
+  statsTrackersRegistry: service('stats-trackers-registry'),
+
+  tagName: 'tr',
+  classNames: ['task-row', 'is-interactive'],
+
+  task: null,
+
+  // Internal state
+  statsError: false,
+
+  enablePolling: computed(() => !Ember.testing),
```

Comment:
Couple of question on this for my own benefit really:

1. Could you use `alias` for this?
2. Does `Ember.testing` ever change once things are running?

Reply:
Alias only works for paths on the object already (e.g., `this.Ember.testing`), so no.

I just tried it and yes, `Ember.testing` can be changed at runtime, but it never happens from framework code.